operator-common/src/main/java/io/strimzi/operator/common/operator/resource/RoleBindingOperator.java

```diff
@@ -76,6 +83,13 @@ public String toString() {
         }
     }
 
+    private static String labelsToString(Labels labels) {
```

Comment:
The current version of the Fabric8 Kubernetes client AFAIK already supports Cluster Roles, Cluster Role Bindings, Role Bindings etc. So I wonder if it would be easier to just replace the workaround operators with the actual operators. That would make handling labels much easier.